tests/Integration/Behaviour/Features/Scenario/Tax/tax_management.feature

```diff
@@ -0,0 +1,56 @@
+@reset-database-before-feature
+#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s tax
+Feature: Manage tax
+  As a employee
+  I must be able to correctly add, edit and delete tax
+
+  Scenario: Adding new tax
+    When I add new tax "tax-1" with following properties:
+      | name         | my custom tax 500 |
+      | rate         | 0.5               |
+      | is_enabled   | 1                 |
+    Then tax "tax-1" name in default language should be "my custom tax 500"
+    And tax "tax-1" rate should be 0.500
+    And tax "tax-1" should be enabled
+
+  Scenario: Editing tax
+    When I edit tax "tax-1" with following properties:
+      | name         | my custom tax 300 |
+      | rate         | 0.15              |
+      | is_enabled   | 0                 |
+    Then tax "tax-1" name in default language should be "my custom tax 300"
+    And tax "tax-1" rate should be 0.150
+    And tax "tax-1" should be disabled
+
+  Scenario: Editing tax when providing only changeable property should be allowed
+    When I edit tax "tax-1" with following properties:
+      | name        | tax for fun       |
+    Then tax "tax-1" name in default language should be "tax for fun"
+    And tax "tax-1" rate should be 0.150
+    And tax "tax-1" should be disabled
+
+  Scenario: Toggling tax status
+    Given tax "tax-1" should be disabled
+    When I toggle tax "tax-1" status
+    Then tax "tax-1" should be enabled
+
+  Scenario: Disabling taxes status in bulk action
+    Given taxes with ids: "1,2,3" exists
+    And taxes with ids: "1,2,3" should be enabled
+    When I disable taxes with ids: "1,2,3"
+    Then taxes with ids: "1,2,3" should be disabled
+
+  Scenario: Enabling taxes status in bulk action
+    And taxes with ids: "1,2,3" should be disabled
+    When I enable taxes with ids: "1,2,3"
+    Then taxes with ids: "1,2,3" should be enabled
+
+  Scenario: Deleting tax
```

Comment:
You can mix some actions together to avoid having too small scenarios 😉 it's authorized as it's integration tests. It's actually interesting to see whether a test works well when paired with another. So we can for example merge the "Toggling tax status" with the "Deleting tax" (run one after the other) or "Disabling taxes status in bulk action" with "Deleting taxes in bulk action"

We can also aim for more user-oriented scenario titles:
`Deleting taxes in bulk action` => `Deleting multiple taxes from a BO listing`